src/js/common/schemaform/FormApp.jsx

```diff
@@ -62,15 +63,16 @@ export default class FormApp extends React.Component {
 
 
     return (
-      <div className="row">
-        <Element name="topScrollElement"/>
-        <div className="usa-width-two-thirds medium-8 columns">
-          {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
-          {content}
-        </div>
-        <div className="usa-width-one-third medium-4 columns show-for-medium-up">
+      <div>
+        <div className="row">
+          <Element name="topScrollElement"/>
+          <div className="usa-width-two-thirds medium-8 columns">
+            {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
+            {content}
+          </div>
+          <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
         </div>
-        <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
+        {formConfig.getHelp && formConfig.getHelp(phoneNumber)}
```

Comment:
It looks like the info for hca and edu differs by more than the phone number (there's some initial text that's different).

I wonder if we should make a default AskVAQuestions component that takes a form specific component as a child. It looks like the bottom paragraph will be the same across the forms. It's the just the top one that can vary.

So AskVAQuestions could be:

```
function AskVAQuestions(props) {
  return (
    <div className="row">
      <div className="usa-width-two-thirds medium-8 columns">
        <div className="ask-va-questions help-sidebar-box">
          <h2 className="help-h2-border help-h2">Need Help?</h2>
          {props.children}
          <p className="talk">To report a problem with this form,<br/>
          call the Vets.gov Technical Help Desk:</p>
          <p className="phone-number">
            <a href="tel:+1-855-574-7286">855-574-7286</a><br/>
            TTY: <a href="tel:+1-855-574-7286">855-574-7286</a><br/>
            Monday - Friday, 8:00 a.m. - 8:00 p.m. ET
          </p>
        </div>
      </div>
    </div>
  );
}
```
And then an hca specific component would have this content:

```
<p className="talk">Enrollment or Eligibility questions:</p>
<p className="phone-number">
  <a href="tel:+877-222-8387">877-222-8387</a><br/>
  Monday - Friday, 8:00 a.m. - 8:00 p.m. ET<br/>
  TTY: <a href="tel:+1-800-829-4833">800-829-4833</a>
</p>
```

And in FormPage you could have:

```
// towards the beginning of the render function
const GetFormHelp = formConfig.getHelp;
...
// In the jsx being returned
<AskVAQuestions><GetFormHelp/></AskVAQuestions>
```